dts/bindings/clock/nuvoton,numaker-scc.yaml

```diff
@@ -47,3 +47,8 @@ properties:
     type: int
     description: |
       Configure core clock (HCLK)
+
+  pd-mode:
+    type: int
+    description: |
+      Configure power down mode
```

Comment:
Maybe expand on the description on this one, mention that this is one of the `NUMAKER_CLK_PMUCTL_PDMSEL_` macros. Also are these the same on all devices in the family? Maybe you could use an an enum instead.

Reply:
Maybe use a string instead with the enum:
```
type: string
enum:
  - "pd"
  - "llpd"
  - "fwpd"
description: |
  pd - normal power down
```

Reply:
Thanks of your advice, it's good to mention to choose one of NUMAKER_CLK_PMUCTL_PDMSEL_XXX from numaker_xxx_clock.h
It's not the same on all devices, so keep int is better.

Reply:
I see, in this case, you should split the file `numaker_m46x_clock.h` into the specific chips you support, this ensures only valid settings for that chip are available :)

You will have:
`numaker_m461_clock.h`
`numaker_m462_clock.h`
...
where `numaker_m461_clock.h` only includes defines valid for that chip :)

Reply:
@bjarki-andreasen ,
m46x series have the same power down modes, so current numaker_m46x_clock.h is general for m46x series.
I mentioned all devices is meaning to include other chip series, like as m2l31x series with SPD0 to 2 and DPD0 to 1 modes, I will add these modes into numaker_m2l31x_clock.h in next PR.
Appreciate your advice.